roles/openshift_logging_elasticsearch/defaults/main.yml

```diff
@@ -37,6 +37,9 @@ openshift_logging_elasticsearch_storage_group: '65534'
 
 openshift_logging_es_pvc_prefix: "{{ openshift_hosted_logging_elasticsearch_pvc_prefix | default('logging-es') }}"
 
+# config the es plugin to write kibana index based on the index mode
+openshift_logging_elasticsearch_kibana_index_mode: "shared_ops"
```

Comment:
I don't think we can just change the default without migrating dashboards users have already created.  As they'll be lost entirely.

I think it has to be opt-in until we have a story on how to handle such a transition.